documentation/api/io.strimzi.api.kafka.model.CruiseControlSpec.adoc

```diff
@@ -140,19 +140,17 @@ You specify capacity limits for Kafka broker resources in the `brokerCapacity` p
 They are enabled by default and you can change their default values.
 Capacity limits can be set for the following broker resources:
 
+* `cpuCores`        - CPU resource in milliCPU or CPU cores (Default: 1)
 * `inboundNetwork`  - Inbound network throughput in byte units per second (Default: 10000KiB/s)
 * `outboundNetwork` - Outbound network throughput in byte units per second (Default: 10000KiB/s)
 
 For network throughput, use an integer value with standard Kubernetes byte units (K, M, G) or their bibyte (power of two) equivalents (Ki, Mi, Gi) per second.
 
-NOTE: Disk and CPU capacity limits are automatically generated by Strimzi, so you do not need to set them.
+NOTE: Disk capacity limits are automatically generated by Strimzi, so you do not need to set them.
 
-[NOTE]
-====
-In order to guarantee accurate rebalance proposal when using CPU goals, you can set CPU requests equal to CPU limits in `Kafka.spec.kafka.resources`.
+NOTE: CPU capacity limits are automatically generated by Strimzi when you set CPU requests equal to CPU limits in `Kafka.spec.kafka.resources`.
 That way, all CPU resources are reserved upfront and are always available.
 This configuration allows Cruise Control to properly evaluate the CPU utilization when preparing the rebalance proposals based on CPU goals.
```

Comment:
Can we somehow join both notes into one? I think it looks a bit weird to have the two notes right after each other.